susemanager/susemanager.spec

```diff
@@ -177,10 +184,11 @@ Bash completion for SUSE Manager CLI tools
 
 # Fixing shebang for Python 3
 %if 0%{?build_py3}
-for i in `find . -type f`;
+for i in `find . -type f -not -name 'mgr-salt-ssh'`;
 do
     sed -i '1s=^#!/usr/bin/\(python\|env python\)[0-9.]*=#!/usr/bin/python3=' $i;
 done
+sed -i '1s=^#!/usr/bin/python3=#!/usr/bin/%{use_python}' src/mgr-salt-ssh
```

Comment:
~~~
[   97s] + sed -i '1s=^#!/usr/bin/python3=#!/usr/bin/python311' src/mgr-salt-ssh
[   97s] sed: -e expression #1, char 43: unterminated `s' command
~~~

Reply:
Whoops. `unterminated `s' command gets me all the time` :sweat_smile:. Thanks for trying, I'll push a fix